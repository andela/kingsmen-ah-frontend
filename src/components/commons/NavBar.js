import React from 'react';
import { Link } from 'react-router-dom';
import './index.scss';


function NavBar() {
  return (
    <div className="bg-blue-700">
      <div className="container mx-auto max-w-6xl overflow-y-scroll">
        <div className="flex items-center justify-center text-white h-full">
          <ul className="mx-4">
            <li className="flex items-center text-gray-400 h-12 text-sm uppercase capitalize">
              <Link to="/tags" className="active mx-3 text-white">Home</Link>
              <Link to="/tags" className="mx-3">Family</Link>
              <Link to="/tags" className="mx-3">Family</Link>
              <Link to="/tags" className="mx-3">Family</Link>
              <Link to="/tags" className="mx-3">Family</Link>
              <Link to="/tags" className="mx-3">Family</Link>
              <Link to="/tags" className="mx-3">Family</Link>
              <Link to="/tags" className="mx-3">Family</Link>
              <Link to="/tags" className="mx-3">Family</Link>
            </li>
          </ul>
        </div>
      </div>
    </div>
  );
}

export default NavBar;
