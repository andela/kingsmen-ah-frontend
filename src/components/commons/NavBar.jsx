import React from 'react';
import './index.scss';

function NavBar() {
  return (
    <div className='border-radius-none bg-blue-700'>
      <div className='container mx-auto overflow-y-scroll'>
        <div className='flex items-center justify-left text-white h-full'>
          <ul>
            <li className='flex items-center text-gray-400 h-12 text-sm uppercase capitalize'>
              <div to='/' className='active mx-3 text-white'>
                Home
              </div>
              <div className='mx-3'>Family</div>
              <div className='mx-3'>Andela</div>
              <div className='mx-3'>People</div>
            </li>
          </ul>
        </div>
      </div>
    </div>
  );
}

export default NavBar;
