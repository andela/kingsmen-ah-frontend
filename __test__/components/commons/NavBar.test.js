import React from 'react';
import NavBar from '@components/commons/NavBar';


describe('<NavBar />', () => {
  it('should render the NavBar component correctly', () => {
    const shallowWrapper = shallow(<NavBar />);

        expect(toJson(shallowWrapper)).toMatchSnapshot();
        expect(shallowWrapper.find('NavBar')).toBeTruthy();
  });
});
