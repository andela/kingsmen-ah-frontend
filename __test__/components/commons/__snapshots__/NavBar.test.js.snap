// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should render the NavBar component correctly 1`] = `
<div
  className="bg-blue-700"
>
  <div
    className="container mx-auto max-w-6xl overflow-y-scroll"
  >
    <div
      className="flex items-center justify-center text-white h-full"
    >
      <ul
        className="mx-4"
      >
        <li
          className="flex items-center text-gray-400 h-12 text-sm uppercase capitalize"
        >
          <Link
            className="active mx-3 text-white"
            to="/tags"
          >
            Home
          </Link>
          <Link
            className="mx-3"
            to="/tags"
          >
            Family
          </Link>
          <Link
            className="mx-3"
            to="/tags"
          >
            Family
          </Link>
          <Link
            className="mx-3"
            to="/tags"
          >
            Family
          </Link>
          <Link
            className="mx-3"
            to="/tags"
          >
            Family
          </Link>
          <Link
            className="mx-3"
            to="/tags"
          >
            Family
          </Link>
          <Link
            className="mx-3"
            to="/tags"
          >
            Family
          </Link>
          <Link
            className="mx-3"
            to="/tags"
          >
            Family
          </Link>
          <Link
            className="mx-3"
            to="/tags"
          >
            Family
          </Link>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
